src/librbd/Operations.cc

```diff
@@ -1318,6 +1318,12 @@ int Operations<I>::update_features(uint64_t features, bool enabled) {
     lderr(cct) << "cannot update immutable features" << dendl;
     return -EINVAL;
   }
+
```

Comment:
Nit: if ```!enabled```, it might be nicer to just mask out ```RBD_FEATURE_FAST_DIFF``` so that attempting to disable fast-diff doesn't implicitly disable object-map.

Reply:
ok, I see. Thanks for the clarification.